dias/config_loader.py

```diff
@@ -207,13 +242,55 @@ def _import_analyzer_class(self, name):
 
     # Data model
     def __getitem__(self, key):
+        """
+        Read a variable from the dias config.
+
+        Parameters
+        ----------
+        key : str
+            Name of the config variable
+
+        Returns
+        -------
+        The value of the config variable.
+        """
```

Comment:
These four functions are just the data model for the task queue.  You shouldn't be this explicit:  these are the standard way of implementing square brackets, &c. and we don't need to explain how that works here.   There should be a section at the top explaining the data model (see above) and then here, just say:
```
Implements `self[key]`
```
for those interested in the implementation.  (Note: since `key` can be a slice, this function can return multiple tasks or none at all.)

Reply:
I used
Return `self[key]`